Dockerfile

```
@@ -0,0 +1,9 @@
+# syntax=docker/dockerfile:1
+FROM python:3
+
+ENV PYTHONUNBUFFERED=1
+WORKDIR /code/nrm_django
+
+COPY . /code/
```

It seems strange to copy the whole repo into /code/ when we just use the nrm_django part of it. I know there isn't much else in there right now, but I can imagine someday getting other stuff in the root of the repo that we don't need in our image:

```suggestion
COPY ./nrm_django /code/nrm_django
```